csharp-selenium-webdriver-sample/CSharpSeleniumWebdriverSample.csproj

```diff
@@ -1,10 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
It would be nice to see comments added to `azure-pipelines.yml` as well, similar to the other sample (but if you'd like to keep that to a separate PR, that's fine)